grader_labextension/grader_labextension/__init__.py

```diff
@@ -1,4 +1,12 @@
-from grader_labextension.api_handler import setup_handlers
+import asyncio
+
+from tornado.httpclient import HTTPClientError
+
+from grader_labextension.registry import HandlerPathRegistry
+from grader_labextension.handlers.base_handler import HandlerConfig
+from traitlets.config.loader import Config
+
+from grader_service.request import RequestService
 
 
```

Comment:
So the only thing I was wondering is how we are loading config as opposed to the version proposed by @meffmadd, [loc](https://github.com/TU-Wien-dataLAB/Grader-Service/blob/06dcbd08d959300be4297b8823e72358668fd1d7/grader_labextension/grader_labextension/__init__.py#L20-L23) 

Were we not trying to ensure that we were loading the config in production using this method?